### PR TITLE
#94 Changed travis.yml to support django>2.0

### PR DESCRIPTION
**Pull request by HargovindArora**
#94 I just changed few lines of code here and most of the work I think was already present there to support Django>2.0. @nemesisdesign or  @rohitasrk Please review this and tell me if this was fine! :)

### CODE REVIEW

**Comment by nemesisdesign** [comment]
Seems was already done during the GCI: https://github.com/openwisp/django-freeradius/commit/cbf4f6f3f931330b12fe9bf0934c3fc1fe45df1c#diff-b4ef698db8ca845e5845c4618278f29a

@HargovindArora please change your commit message by adding a subsequent line that closes the issue @rohithasrk has created, eg:

```
[ci] Changed travis.yml to support django 2.0 #94

Closes #94
```

**Comment by rohithasrk** [comment]
Also @HargovindArora, You should modify the `requirements-test.txt` as well.

File: .travis.yml [hunk 1]
```diff
@@ -13,12 +13,12 @@ branches:
...
-   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
-   - DJANGO="django>=1.11,<1.12" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
-   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
    - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
    - DJANGO="django>=2.0,<2.1" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
-   - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1   
+   - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+   - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
+   - DJANGO="django>=2.0,<2.1" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
+   - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
```
[the remark this answers is not in the document]
Comment (HargovindArora) [reply .travis.yml]: @rohithasrk So, should I remove the last three lines in this?

[the remark this answers is not in the document]
Comment (rohithasrk) [reply .travis.yml]: Yes. You may proceed.

[the remark this answers is not in the document]
Comment (HargovindArora) [reply .travis.yml]: Okay sure.

**Comment by HargovindArora** [comment]
@rohithasrk Is this looking good now?

**Pull request merged**